config/samples/redhatcop_v1alpha1_jwtoidcauthengineconfig.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: redhatcop.redhat.io/v1alpha1
+kind: JWTOIDCAuthEngineConfig
+metadata:
+  name: azure-oidc-config
+spec:
+  authentication: 
+    path: kubernetes
+    role: policy-admin
+  path: oidc/azuread-oidc/
+  OIDCClientID: "123456-1234-1234-1234-123456789"
+  OIDCCredentials:
+    secret: 
+      name: oidccredentials
+    usernameKey: oidc_client_id
+    passwordKey: oidc_client_secret
+  OIDCDiscoveryURL: "https://login.microsoftonline.com/123456789-1234-1234-1234-123456789/v2.0"
```

Comment:
Thnx for the feedback.
In this case, OIDCClientID would be an Azure AD SP client_id, and the ID in the OIDCDiscoveryURL would be an Azure Tenant (Directory) ID. Since from a Security perspective I can't say much, I found this RFC : https://www.rfc-editor.org/rfc/rfc6749#section-2.2 , which says : 
`The client identifier is not a secret; it is exposed to the resource owner and MUST NOT be used alone for client authentication.`
That would be the same even for Azure Tenant ID. But what would you suggest ?

Reply:
given that this is just an example, can you just replace the tenant it with all zeros?

Reply:
done.

Reply:
what did you change, I still see your client id

Reply:
I changed only the Azure Tenant ID as you suggested. SP Client ID is changed too.